### PR TITLE
Add Semaphore v2 CI example 266

### LINKED ISSUE #266: Finish Semaphore V2 CI
Already working, need to show how to do
- Test Runner binary caching
- recording tests
- running tests in parallel

### PR DESCRIPTION
**Pull request by bahmutov**
- closes #266 
- both basic and parallel configs

### CODE REVIEW

**Comment by bahmutov** [comment]
Missing fields on Dashboard and extract `ci-build-id` automatically is in https://github.com/cypress-io/cypress/issues/4671

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Codeship Pro | [ ![Codeship Pro CI](https://app.codeship.com/projects/8d6a20c0-b
 GitLab | [![GitLab CI](https://gitlab.com/cypress-io/cypress-example-kitchensink/badges/master/pipeline.svg)](https://gitlab.com/cypress-io/cypress-example-kitchensink/commits/master) | [basic/.gitlab-ci.yml](basic/.gitlab-ci.yml) | [.gitlab-ci.yml](.gitlab-ci.yml)
 Heroku CI | | [basic/app.json](basic/app.json) |
 Jenkins | | [basic/Jenkinsfile](basic/Jenkinsfile) | [Jenkinsfile](Jenkinsfile)
-Semaphore v2 | [![Semaphore CI](https://semaphoreci.com/api/v1/cypress-io/cypress-example-kitchensink/branches/master/badge.svg)](https://semaphoreci.com/cypress-io/cypress-example-kitchensink) | [.semaphore/semaphore.yml](.semaphore/semaphore.yml)
+Semaphore v2 | [Project dashboard](https://cypress-io.semaphoreci.com/projects/cypress-example-kitchensink) | [basic/.semaphore.yml](basic/.semaphore.yml) | [.semaphore/semaphore.yml](.semaphore/semaphore.yml)
```
Comment (jennifer-shehane) [inline README.md]: Shouldn't you link Semaphore v1 here too?

Comment (bahmutov) [reply README.md]: but since Semaphore v1 instructions are invisible to people, I think we should just retire it

**Comment by bahmutov** [comment]
by the way, I have sent an email to Semaphore team asking if people can view the dashboard for a public project without logging in, and if there is a status badge, I think they are not ready, since this is still new version of their CI

**Pull request merged**